README.md

```diff
@@ -1,8 +1,8 @@
-# Context Template Repository _(context-template)_
+# Ed25519Signature2018 Suite JSON-LD Context _(ed25519-signature-2018-context)_
 
-<!-- [![NPM Version](https://img.shields.io/npm/v/context-template.svg?style=flat-square)](https://npm.im/context-template) -->
+<!-- [![NPM Version](https://img.shields.io/npm/v/ed25519-signature-2018-context.svg?style=flat-square)](https://npm.im/ed25519-signature-2018-context) -->
 
-> A GitHub template repository for JSON-LD contexts.
+> JSON-LD context for the Ed25519Signature2018 suite.
```

Comment:
Not necessarily true, more like -- JSON-LD Context for the Ed25519 2018 Cryptosuite.

Reply:
Updated, thanks.